### DISCUSSION TITLE
HTML hidden attribute overridden by ui class?

### DISCUSSION THREAD

**Discussion by ultimodruido** [fomantic/Fomantic-UI#1361] [General] [answered]
It seems that the "hidden" attribute is not considered by the browser when a tag has the "ui" class.

Examples:
This element is visible
`<div class="ui placeholder segment" hidden><h1>Content #1</h1></div>`
Removing only the "ui" class makes this element disappear.
`<div class="placeholder segment" hidden><h1>Content #1</h1></div>`

Tested in Opera 67

**Comment by lubber-de** [comment, marked as answer]
Yes, it's because any `ui` element changes the `display` css value which itself overrides any existing hidden-property

By adding the following small line of CSS code, you should be able to force the hidden-property to gain more respect
```css
[hidden] { display:none !important;}
```
